docs/driver-only-builds.md

```diff
@@ -240,3 +241,26 @@ removing builtin support (i.e. `MBEDTLS_DHM_C`).
 ### Limitations
 Support for deterministic derivation of a DH keypair
 (i.e. `PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_DERIVE`) is not supported.
+
+AEADs
+-----
+
+It is possible to have all AEADs operations provided only by a driver.
+
+More precisely you can:
+- enable desired PSA algorithm(s) and key type(s):
+  - `PSA_WANT_ALG_[CCM|GCM]` with `PSA_WANT_KEY_TYPE_[AES|ARIA|CAMELLIA]`
+  - `PSA_WANT_ALG_CHACHA20_POLY1305` with `PSA_WANT_KEY_TYPE_CHACHA20`;
+- enable `MBEDTLS_PSA_ACCEL_xxx` symbol(s) which correspond to the
+  `PSA_WANT_xxx` of the previous step;
+- disable builtin support of `MBEDTLS_[CCM|GCM|CHACHAPOLY]_C` algorithms and
+  key types `MBEDTLS_[AES|ARIA|CAMELLIA|CHACHA20]_C` for AEADs which are
+  accelerated.
+
+In such a build all AEADs operations requested through the PSA Crypto API
+(including those in TLS and X.509) will be performed by the provided driver.
+Of course direct calls to the disabled builtin modules
+(ex: `mbedtls_ccm_init()`, etc) won't be possible.
```

Comment:
Also, once you've disabled `AES_C` you won't get PEM descrytion, or PKCS5/15 or their use in in PKparse. (And perhaps others I'm forgetting, we should double-check what's disabled in the `all.sh` component and what's ignored in `analyze_coverage.py`.)

So, I think we need to change the structure like:
1. if all AEAD algs are accelerated, you can disable the AEAD modules (TODO: list them) and everything will work.
2. if additionally you don't need unauthenticated ciphers, you can disable `AES_C` etc and also `CIPHER_C` and everything will work except for PEM etc. (TODO: complete list - for now this even includes `CTR_DRBG_C`).